fdbrpc/ActorFuzz.actor.cpp

```diff
@@ -140,6 +141,7 @@ ACTOR Future<int> actorFuzz1(FutureStream<int> inputStream, PromiseStream<int> o
 }
 
 ACTOR Future<int> actorFuzz2(FutureStream<int> inputStream, PromiseStream<int> outputStream, Future<Void> error) {
+	state int ifstate = 0;
```

Comment:
These don't cause any issues with the open for IDE CI?

Reply:
Assuming the answer is that they don't, this looks good to me

Reply:
They cause warnings and initially I wanted to get rid of warnings as well. But then I realaized that cleaning up all the warnings is going to be a lot more work than I can feasibly do right now